features/step_definitions/assertion_steps.rb

```diff
@@ -5,3 +5,11 @@
 Then('I should not see {string}') do |content|
     expect(page).not_to have_content content
 end
+
+Then('show me the page') do
+    save_and_open_page
+end
+
+Then('I click on logo') do
```

Comment:
This test is associated with the scenario we take out so please take this out as well

Reply:
Done